### ISSUE TITLE
Landing Page

### ISSUE THREAD

**Issue by adrianolisboa** [fatiapp/discussions#4]
Considering that we now have **fatiapp.org** registered, does it make sense to have a landing page explaining what will be the app and how to contribute ?

BTW, at this start point it will be opened for everyone to contribute or just us ?

**Comment by fernandoalmeida** [comment]
yes! make sense! :+1:

We really need to describe what exactly we want in the project (regardless of a landing page).

I believe we will have many things beyond the financial management. For example, I am documenting the concept of "personal cycles", that I would like to register to have a history, helping some decision making and so on.

Maybe each one of us could list what we expect to have in the project and further we merge this lists and discuss about it.

What do you think?

**Comment by adrianolisboa** [comment]
Yeah, make sense. I'll think about what I expect about the project.
